bin/dev

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env sh
+
+# Default to port 3000 if not specified
+export PORT="${PORT:-3000}"
+
+if command -v overmind &> /dev/null
+then
+  overmind start -f Procfile.dev "$@"
+  exit $?
+fi
+
+if ! gem list foreman -i --silent; then
```

Comment:
Maybe we can add `foreman` to the development group in `Gemfile`? Wouldnt it be even simpler?

Reply:
Recommendation is not to - https://github.com/ddollar/foreman/wiki/Don't-Bundle-Foreman

Reply:
Good point. I fear that invoking `gem list` for each invocation is not very performant, maybe we could simply check if the command exists?

Reply:
You know what -- I don't actually know the full history here of checking via `gem list` vs a more typical shell command check. If I had to guess, there's a history here of different distribution channels (homebrew, a node package? a go package? the ruby gem, etc) and I bet this approach is to dial in on the gem one specifically).

That said, in researching this I found some issues/PRs on the various rails gem packages that install this script, and they have a slightly more targetted/performant line to find the gem, which I've just put in place here. Running this locally (with either overmind or foreman) there truly is like ~zero delay felt by this check, it goes pretty much immediately into log output for the various commands. Obviously if it was doing the one-time install for you there'd be a small delay with some output to see what was being done.